build/bin/network-validator.go

```diff
@@ -86,11 +87,29 @@ func TestEndpoints(config reachabilityConfig) {
 }
 
 func ValidateReachability(host string, port int) error {
-	endpoint := fmt.Sprintf("%s:%d", host, port)
+	var endpoint string
+
+	httpClient := http.Client{
+		Timeout: *timeout * time.Second,
```

Comment:
I believe you can skip the `* time.Second` here given the type of `timeout`

Reply:
+1